third_party/mbedtls/repo.patched/include/mbedtls/ssl_ticket.h

```diff
@@ -92,7 +92,7 @@ void mbedtls_ssl_ticket_init( mbedtls_ssl_ticket_context *ctx );
  *                  Recommended value: 86400 (one day).
  *
  * \note            It is highly recommended to select a cipher that is at
- *                  least as strong as the the strongest ciphersuite
+ *                  least as strong as the strongest ciphersuite
```

Comment:
We should probably leave this typo as-is, since we pull this from mbedtls directly.